### PR TITLE
Fix JENKINS-56016 by documenting behaviour

### PR DESCRIPTION
**Pull request by papanito**
Fix JENKINS-56016 by documenting behaviour

### CODE REVIEW

**Comment by papanito** [comment]
@Wadeck can you merge the changes. I have no write permissions - at least there is no merge button

**Comment by Wadeck** [comment]
@papanito I have no write permission here, you can check this repo for the rights: https://github.com/jenkins-infra/repository-permissions-updater/blob/master/permissions/plugin-pipeline-input-step.yml ;) 

@dwnusbaum when you have time, it could be good to merge this PR, thx :)

File: src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <div>
     User IDs and/or <em>external</em> group names of person or people permitted to respond to the input, separated by ','.
-    Spaces will be trimmed. This means that "alice, bob, blah " will be equivalent to "alice,bob,blah".
+    Spaces will be trimmed. This means that "alice, bob, blah " will be equivalent to "alice,bob,blah".<br>
+    <strong>If you are an administrator the restriction does not apply and you are permitted to respond regardless if your are in the list or not.</strong>
```
Comment (dwnusbaum) [inline src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html]: Thanks for filing a PR! I might reword this a bit, but I'd be interested to hear what @bitwiseman thinks would work well here. Here's how I would phrase it:

```suggestion
    <strong>Note: Jenkins administrators are able to respond to the input regardless of the value of this parameter.</strong>
```

Comment (bitwiseman) [reply src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html]: Yup, what @dwnusbaum said.

File: src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <div>
     User IDs and/or <em>external</em> group names of person or people permitted to respond to the input, separated by ','.
-    Spaces will be trimmed. This means that "alice, bob, blah " will be equivalent to "alice,bob,blah".
+    Spaces will be trimmed. This means that "alice, bob, blah " will be equivalent to "alice,bob,blah".<br>
```
Comment (bitwiseman) [inline src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/InputStep/help-submitter.html]:
```suggestion
    Spaces will be trimmed. This means that "alice, bob, blah " is equivalent to "alice,bob,blah".<br>
```

**Comment by papanito** [comment]
Thanks guys!

**Pull request merged**